src/account/account.repository.knex.integration.test.ts

```diff
@@ -1,4 +1,4 @@
-import { beforeAll, describe, expect, it, vi } from 'vitest';
```

Comment:
Sorry for the messy diff in this test

The test i've added for the functionality change is the last one in the diff

The rest of the changes is just some refactoring to remove code duplication and ensure we have fresh data for each test as we were unintentionally relying on data from previous test runs 😬

Reply:
Lol this diff is insane